app/models/userModel.js

```diff
@@ -0,0 +1,41 @@
+import pool from '../db';
+import helpers from '../middleware/helpers';
+
+export default class UserModel {
+  static async create(req, res) {
+    const hashedPassword = helpers.hashPassword(req.body.password);
+    const newUserQuery = `INSERT INTO users(firstname, lastname, email, type, password) VALUES($1, $2, $3, $4, $5) 
+                          RETURNING user_id, firstname, lastname, email, type`;
+    const { firstname, lastname, email, type } = req.body;
+    const values = [firstname.trim(), lastname.trim(), email.trim(), type.trim(), hashedPassword];
+    try {
+      const { rows } = await pool.query(newUserQuery, values);
+      return rows;
+    } catch (error) {
+      if (error.routine === '_bt_check_unique') {
+        return res.status(400).send({
+          status: 400,
+          error: `User with '${req.body.email}' already exists`,
```

Comment:
change this message to `user already exists` no need to include the email

Reply:
okay